rakelib/test.rake

```diff
@@ -16,6 +16,13 @@ namespace "test" do
     RSpec::Core::Runner.run(["--fail-fast", *Rake::FileList["spec/**/*.rb"]])
   end
 
+  task "all-plugins" => [ "bootstrap","plugin:install-all" ] do
+    require "logstash/environment"
+    gem_home = LogStash::Environment.logstash_gem_home
+    pattern = "#{gem_home}/logstash-*/spec/{input,filter,codec,output}s/*_spec.rb"
+    sh "#{LogStath::Environment::LOGSTASH_HOME}/bin/logstash rspec #{pattern}"
```

Comment:
When supplying any arguments it removes all defaults.
Also devutils is not in use for this part.